starter-code/routes/drones.js

```diff
@@ -1,15 +1,25 @@
 const express = require('express');
 
 // require the Drone model here
+const Drone = require('../models/drone');
 
 const router = express.Router();
 
 
-router.get('/drones', (req, res, next) => {
+router.get('/', (req, res, next) => {
   // Iteration #2
+  Drone.find({}, (err, droneData) => {
+    if (err) { return next(err) }
+
+    res.render('drones/index', {
+      droneData: droneData
```

Comment:
This looks good. All you need to do now for this iteration is to show them in the ejs file. Remember the difference between `<%=  %>` and `<%  %>` =]